src/index.ts

```diff
@@ -662,9 +662,16 @@ function FlatpickrInstance(
       dayElement = createElement<DayElement>(
         "span",
         "flatpickr-day " + className,
+        ''
+      ),
+      innerDayElement = createElement<DayElement>(
+        "span",
```

Comment:
We need another html element to have different sizes of elements.
![Screen Shot 2020-10-22 at 7 52 16 PM](https://user-images.githubusercontent.com/2461485/96910657-1d126600-14a0-11eb-9462-d10f7c3a40c4.png)